doc/release-notes.md

```diff
@@ -102,6 +102,22 @@ will trigger BIP 125 (replace-by-fee) opt-in. (#11413)
   option `-deprecatedrpc=banscore` is used. The `banscore` field will be fully
   removed in the next major release. (#19469)
 
+- The `getpeerinfo` RPC no longer returns the `inbound` or `addnode` fields by
+  default. These two fields will be fully removed in the next major release.
+  They can be accessed with the configuration options
+  `-deprecatedrpc=getpeerinfo_inbound` and `-deprecatedrpc=getpeerinfo_addnode`
+  respectively. However, it is recommended to instead use the `connection_type`
+  field, which will indicate whether the peer is inbound, addnode(manual), or
+  another type of connection.
+
+- The `getpeerinfo` RPC now returns a `connection_type` field. This indicates
+  the type of connection established with the peer. It will return one of six
+  options: 1. outbound full relay (default automatic connections) 2. block
+  relay only (connections that only relay blocks, not transactions or
+  addresses) 3. inbound (connection initiated by the peer) 4. manual (peers
+  added via addnode or -connect configuration param) 5. feeler / 6. addrfetch
+  (both short lived automatic connections used for robustness).
```

Comment:
nit: instead of adding extended documentation to the release notes, the documentation can be added to the rpc itself. Affected/interested users can call `help rpc_method` or `--help` if needed.

Reply:
okay, removed the specifics. better?